Cargo.toml

```diff
@@ -119,6 +127,9 @@ syn = "2.0.0"
 tempfile = "3.6.0"
 thiserror = "1.0.35"
 tokio = "1.18.0"
+tokio-stream = "0.1.14"
+tonic = "0.10.2"
+tower = "0.4.13"
```

Comment:
I wonder if we can lower the minimum required version to 0 for these dependencies.

Reply:
Lowered `tokio-stream` and removed the other two since those are now imported from the `astarte-message-hub-proto` crate.